### ISSUE TITLE
Unable to get updates?

### ISSUE THREAD

**Issue by fuzzy76** [kaimallea/csgo#26]
How is this image supposed to get updates? I see you include something called Updater, but I have no idea what that is, and your readme doesn't mention how updates happen (or are supposed to be done).

I see this:
```
csgo_1        | Updating server using Steam.
csgo_1        | ----------------------------
csgo_1        | /home/steam/csgo/srcds_run: line 301: ./steam.sh: No such file or directory
csgo_1        | ----------------------------
csgo_1        | Setting breakpad minidump AppID = 740
csgo_1        | Using breakpad crash handler
csgo_1        | Forcing breakpad minidump interfaces to load
csgo_1        | Looking up breakpad interfaces from steamclient
csgo_1        | Calling BreakpadMiniDumpSystemInit
csgo_1        | [S_API FAIL] SteamAPI_Init() failed; SteamAPI_IsSteamRunning() failed.
csgo_1        | [S_API FAIL] SteamAPI_Init() failed; unable to locate a running instance of Steam, or a local steamclient.so.
```

Followed by this:

```
csgo_1        | Your server is out of date and will be shutdown during hibernation or changelevel, whichever comes first.
csgo_1        | L 04/01/2020 - 09:31:07: Your server is out of date and will be shutdown during hibernation or changelevel, whichever comes first.
csgo_1        | Host state 5 at Wed Apr  1 09:31:07 2020
csgo_1        |  -- sv_shutdown hibernating server right now.
csgo_1        | L 04/01/2020 - 09:31:07: [META] Loaded 0 plugins (1 already loaded)
csgo_1        | Host_ShutdownServer
csgo_1        | CBaseServer::Shutdown
csgo_1        | CSteam3Server::Shutdown
csgo_1        | NET_SteamGameServer_Shutdown
csgo_1        | SteamGameServer_Shutdown
csgo_1        | L 04/01/2020 - 09:31:08: server_message: "quit"
csgo_1        | L 04/01/2020 - 09:31:08: Log file closed
csgo_1        | L 04/01/2020 - 09:31:08: server_message: "restart"
csgo_1        | SV_ShutdownGameDLL
csgo_1        | Host_Shutdown
csgo_1        | Shutdown function NET_Shutdown() not in list!!!
csgo_1        | NET_Shutdown
csgo_1        | NET_CloseAllSockets
csgo_1        | NET_SteamGameServer_Shutdown
```

**Comment by fuzzy76** [comment]
Restarting the container seemed to install the update. Not closing yet, since I'm not sure if that is how updates are supposed to work or not.

**Comment by kaimallea** [comment]
Hey, thanks for asking. Hopefully I can provide clarity and contribute it to the README.

`Updater` is a plugin that automatically updates SourceMod plugins only. And only those plugins that make use of it.

Re: CSGO updates, it is _supposed_ to update itself, without any intervention from you, via the the use of `-autoupdate`, `-steamdir` and `-steamcmd_script` ([Reference](https://developer.valvesoftware.com/wiki/Command_Line_Options)), that is set by default. If it's not working, then I'll look into why.

Restarting the container will always work because the entrypoint script will attempt to install updates prior to running the game.

**Comment by kaimallea** [comment]
This error that has always existed in the official srcds scripts might be related: `/home/steam/csgo/srcds_run: line 301: ./steam.sh: No such file or directory`

Might have to hotfix Valve's bash script as part of the image

**Comment by kaimallea** [comment]
@fuzzy76 should be fixed now in #28. Apparently Valve shipped their `srcds_run` script with a bug/typo that breaks autoupdates for all games. 🤦‍♂
